doc/tutorial/single_cell_model.rst

```diff
@@ -114,10 +113,10 @@ recording outputs and running the simulation.
     # (7) Run simulation for 30 ms of simulated activity.
     m.run(tfinal=30)
 
-Step **(5)** instantiates the :class:`arbor.single_cell_model<arbor.single_cell_model>`
+Step **(5)** instantiates the :class:`arbor.single_cell_model`
 with our single-compartment cell.
 
-Step **(6)** adds a :meth:`arbor.single_cell_model.probe()<arbor.single_cell_model.probe>`
+Step **(6)** adds a :meth:`arbor.single_cell_model.probe`
```

Comment:
I added the brackets because unfortunately the html rendering is identical to member variable or class.

Reply:
Is it a big problem? The link works and the example shows it's a method.

Reply:
Hmm, I see that now too. Strange, could have sworn it didn't!